README.md

```diff
@@ -101,7 +101,7 @@ To use bbolt as an embedded key-value store, import as:
 ```go
 import bolt "go.etcd.io/bbolt"
 
-db, err := bolt.Open(path, 0666, nil)
+db, err := bolt.Open(path, 0o600, nil)
```

Comment:
Unless I am misunderstanding something should this be `0600`, it looks like an extra character has been added by accident?

Ref: https://pkg.go.dev/os#FileMode

Reply:
"0o600" is a correct octal Integer literal.  `0, 0o, or 0O` are all valid prefix for octal integer literal. FYI. https://go.dev/ref/spec#Integer_literals .

Since all examples (even the golang standard lib) use the prefix `0`, so suggest to change all the example to `0600` as well.

Reply:
Ahh thanks for clarification! Yeah agree I always think of file modes as written in style of `xxxx` i.e. `0600`.

Reply:
I did the two git commands you asked for.

I'm also used to `0600` etc. However, `0o600` is unambiguous and therefore preferable.

As for the std. lib. using `0`, well I consider doing so to be a doc bug.

Reply:
Hi @mark-summerfield - DCO Check is still failing as each commit needs to be signed, I didn't realise this small change was multiple commits, you could either squash your commits and push one signed commit or update both commits to ensure they are signed.

As for `0600` vs `0o600`, my preference would be to stick with `0600` to align with standard go lib examples and as a more recognizable and commonly used approach for expressing file mode.  Will leave it to maintainers to decide if that feedback is blocking for this change.

Reply:
I don't know how to update github commits.

As for `0600`, as I said before the std. docs are at fault for not using `0o600` etc.

Reply:
> As for the std. lib. using 0, well I consider doing so to be a doc bug.

I don't know where is your conclusion coming from. Please convince Golang contributors firstly in Golang community. Before that, please update all the examples using prefix "`0`".

Note that not only golang doc & examples, but also golang lib use prefix `0` for the fileMode.  FYI. [fs.go#L192](https://github.com/golang/go/blob/3e35df5edbb02ecf8efd6dd6993aabd5053bfc66/src/io/fs/fs.go#L192)